spec/message.abnf

```diff
@@ -21,7 +21,7 @@ annotation = (function *(s option)) / reserved / private-use
 literal = quoted / unquoted
 variable = "$" name
 function = (":" / "+" / "-") name
-option = name [s] "=" [s] (literal / variable)
+option = name [[s] "=" [s] (literal / variable)]
```

Comment:
Perhaps it would be clearer to separate the value part (CLDR does this with Unicode Locale Identifiers, which have a similar feature)

```suggestion
option = name [option-value]
option-value = [s] "=" [s] (literal / variable)
```